### DISCUSSION TITLE
Clean fresh install of RaspAP and no internet

### DISCUSSION THREAD

**Discussion by TheDogHusky** [RaspAP/raspap-webgui#1581] [Q&A]
Hey.
Some of you might consider this discussion as a duplicate, but I tried every solution sent in other discussions mentioning my issue and none works.

I made a fresh install using the quick installer, used ethernet to connect to internet (eth0) and used a wifi usb dongle with the correct RTL drivers (got the interface wlan0 working, running on a raspberry pi 1).
I can connect to the network (that I called "Adam Network") and access the dashboard, but not the web. My iPhone told me that this wifi network does not have access to internet.
It correctly gave me an IP address.

 Any ideas?

**Comment by billz** [comment]
Lire le point 3 ici: https://docs.raspap.com/issues/#guidelines

**Reply by TheDogHusky** [reply]
Here is the debug log. Sorry for that mistake.
[raspap_debug.log](https://github.com/RaspAP/raspap-webgui/files/15255504/raspap_debug.log)

**Reply by billz** [reply]
Docker is running on your system. Did you install RaspAP in a Docker container?

**Reply by TheDogHusky** [reply]
Nah as I said, I just ran the quick install as said in the docs

**Reply by billz** [reply]
This is not what I'd call a "fresh" install. It's not clear what (if any) iptables rules were added to the host for Docker. Disable it and reboot. Disable WireGuard and OpenVPN, which are both enabled but not configured.
